gitpod-io/website
# Renamed preview installation to local preview in documentation

## Description
Renamed preview installation to local preview in documentation

## How to test
Check the changes I made and make sure that I changed all the mentions of `preview installation` 

## Release Notes

```release-note
NONE
```


<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2326"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2326"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
